tgg/seqr_scripts/pull_seqr_pedigrees.py

```diff
@@ -29,19 +29,24 @@ def pull_project_peds(session_id: str, projects: set):
             )
             for project_guid in projects:
                 project_guid = project_guid.rstrip()
-                r = s.get(
-                    f"{SEQR_URL}/api/project/{project_guid}/details"
+                family_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/get_families"
                 )
```

Comment:
since this last field is the only thing that changed, maybe it would be helpful to pass in the field to check as an argument? something like 
```suggestion
                family_r = s.get(
                    f"{SEQR_URL}/api/project/{project_guid}/{get_family_url}"
                )
```
get_family_url could also be a constant if it doesn't change that often

Reply:
Do you mean that because the process for getting `family_r` and `individual_r` are similar, we can replace that last bit with the variable? what did you mean by creating an `get_family_url ` arg?

Reply:
ah sorry this wasn't clear! for this, I noticed that the end of the URL changed from `details` to `get_families`. I don't know if this might change again in the future, but the suggestion was to do something like this:
```
get_family_url = "get_families"
...
family_r = s.get(
    f"{SEQR_URL}/api/project/{project_guid}/{get_family_url}"
)
```
so that the only thing we'd have to change in the future is the variable `get_family_url`

Reply:
oh gotcha, we'd have to do that for `get_family_url` and `get_individual_url` right? I'm not sure if the API calls might change in the future, but I think that makes sense to define them as variables